src/FlakeId/Id.cs

```diff
@@ -142,7 +142,10 @@ private void CreateInternal(long timeStampMs = 0)
             long milliseconds = timeStampMs == 0 ? MonotonicTimer.ElapsedMilliseconds : timeStampMs;
             long timestamp = milliseconds & TimestampMask;
             int threadId = Thread.CurrentThread.ManagedThreadId & ThreadIdMask;
-            int processId = s_processId ??= Process.GetCurrentProcess().Id & ProcessIdMask;
+            // int processId = s_processId ??= Process.GetCurrentProcess().Id & ProcessIdMask;
+            if (s_processId is null)
+                s_processId = Process.GetCurrentProcess().Id & ProcessIdMask;
+            int processId = s_processId.Value;

```

Comment:
This is the exact same with extra steps, I'd much prefer to keep the old code.

Reply:
I see. Even then there is a more succinct way of doing it, something along the lines of:

```csharp
s_processId.HasValue ? s_processId.Value : (s_processId = Process.GetCurrentProcess().Id & ProcessIdMask)
```

Now that I'm reading this though, part of me thinks we shouldn't shift the mask into `s_processId` at all, but that's something for later.

Reply:
I know it is three lines instead of one, but this was a clear way of doing it. I don't insist on using the exact lines, feel free to replace it with anything compatible with good old C# 7.3 (the latest language supported by .NET Standard 2.0)